src/Symfony/Component/Debug/Exception/FlattenException.php

```diff
@@ -278,3 +279,35 @@ private function getClassNameFromIncomplete(\__PHP_Incomplete_Class $value)
         return $array['__PHP_Incomplete_Class_Name'];
     }
 }
+
+namespace Symfony\Component\HttpKernel\Exception;
```

Comment:
It can, per https://github.com/symfony/symfony/pull/13206/files#diff-230e5606e6f592c8b8973fc758d112c8R23